README.md

```diff
@@ -530,6 +536,20 @@ All scm materials can have filter object:
   "shallow_clone": true
 }
 ```
+<a name="git-material"/>
+Since GoCD 19.4.0 onwards, you can specify username and password as such
```

Comment:
@vrushaliwaykole Is this `you can` or `you need to`? Especially with `format_version: 5`. If it is, `you can`, then it's probably best to describe what happens if it's defined in both places, the URL and the attributes.

Reply:
@arvindsv It should be `you need to` as we have decided to remove support of username and password from URL. I will update the documentation.